apps/src/templates/studioHomepages/BorderedCallToAction.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import styleConstants from '@cdo/apps/styleConstants';
+import Button from '@cdo/apps/templates/Button';
+import {navigateToHref} from '@cdo/apps/utils';
+
+const BorderedCallToAction = ({
```

Comment:
I find this name much more descriptive, nice change!

Reply:
Shoutout to @madelynkasula for that one